build-system/tasks/compile.js

```diff
@@ -271,6 +271,13 @@ function compile(entryModuleFilenames, outputDir,
       compilerOptions.compilerFlags.define.push('TYPECHECK_ONLY=true');
       compilerOptions.compilerFlags.jscomp_error.push(
           'checkTypes', 'accessControls', 'const', 'constantProperty');
+
+      // TODO(aghassemi): Remove when NTI is the default.
+      if (argv.useNTI) {
```

Comment:
can we use something more idiomatic like `nti`, using the flag already implies you want to use it